recipes/hmat-oss/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.1.2" %}
+{% set variant = "openblas" %}
+
+package:
+  name: hmat-oss
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
```

Comment:
Personally, I don't care, but sometimes it is nice to have this include the package name. It doesn't really affect conda-forge, but it can be nice for local use. For instance, this could be `hmat-oss-{{ version }}.tar.gz`. That way if you end up looking for something in your source cache you can find it more easily and have a better idea of what is there. Again this is not an action item just a tip.